package.json

```diff
@@ -42,11 +42,11 @@
     "@emotion/react": "^11.1.2",
     "@guardian/eslint-plugin-source-react-components": "^9.0.0",
     "@guardian/grid-client": "^1.1.1",
-    "@guardian/libs": "^10.1.1",
+    "@guardian/libs": "^13.1.0",
     "@guardian/node-riffraff-artifact": "^0.3.2",
     "@guardian/source-foundations": "^7.0.1",
```

Comment:
Can we bump `@guardian/source-foundations` too?

Reply:
Yep definitely can do. It looks like both DCR and frontend are on v7 so I guess we'd want to upgrade them at the same time?